nixos/modules/services/misc/gitlab.nix

```diff
@@ -774,6 +775,10 @@ in {
 
           ${pkgs.openssl}/bin/openssl rand -hex 32 > ${cfg.statePath}/gitlab_shell_secret
 
+          if [[ -h '${cfg.statePath}/config/database.yml' ]]; then
+            rm '${cfg.statePath}/config/database.yml'
+          fi
+
           ${if cfg.databasePasswordFile != null then ''
```

Comment:
Not related to this issue, but I noticed that no `database.yml` gets written if the `databasePasswordFile` is not set. So it is impossible to exclusively use `extraDatabaseConfig` to configure the database.
In other modules, it is sometimes easier to ditch the options and put a complete config in the `extraConfig`.
Is that intentional?

Reply:
If that's the case, then that's a bug, but I don't have `databasePasswordFile` set and get a `database.yml` anyway (see the else clause of the if expression)

Reply:
No problem :)